src/theme/colors.ts

```diff
@@ -51,6 +51,10 @@ const commonColors = {
     700: '#C74543',
     800: '#AE3C3B',
     900: '#923231'
+  },
+  darkTeal: {
```

Comment:
For buttons `colorScheme`, we don't need anything else then 500 and 300. Will add the full palette in this PR so that this can be reused.